week2/olya/sentence_validator.py

```diff
@@ -0,0 +1,29 @@
+import re
+
+def sentence_validator(text):
+    """
+    Check  if all sentences in the text are valid
+        Args: string
+        Returns: true or false
+    """
+
+    pattern = "[.?!]"
+    sentences = re.split(pattern, text)
+
+    for sentence in sentences[:-1]:
+        word = sentence.strip()
+        if word[0].islower():
+            return False
+    for sentence in sentences[:-1]:
+        comma_pattern = ",(?=\S)|:"
+        commas = re.split(comma_pattern, sentence)
+        if commas.__len__() != 1:
```

Comment:
Why do you use .__len__() instead of len(). Did you customize it !?

Reply:
No it's not custom, so yes it will good if i use len()